services/api/migrations/20230206190746_mosaic-group.js

```diff
@@ -0,0 +1,32 @@
+function up(knex) {
+    return knex.schema.raw(`
+        CREATE TABLE mosaic_groups (
```

Comment:
@ingalls can we match the terminology that people are used to? I think it's ideal if we call this imagery_source (naip, sentinel) and then followed by mosaics